src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.html

```diff
@@ -22,22 +22,14 @@
           <td class="interactorsColumn"
```

Comment:
```
<td class="interactorsColumn horizontalHeader"
              [class.fixedWidth]="fixedWidth()"
              [style]="ranges.length!=0 ? 'left:3ch':''">
```

Reply:
The CSS file also needs to be updated.

Currently, it is like this, which it is not working.
```
.horizontalHeader {
  min-width: 114px;
  max-width: 114px;
}

.horizontalHeader.fixedWidth {
  width: 230px;
}
```

It works if you do:
```
.horizontalHeader {
  min-width: 114px;
  max-width: 114px;
}

.horizontalHeader.fixedWidth {
  width: 230px;
  min-width: 230px;
  max-width: 230px;
}
```